src/sagemaker/tensorflow/estimator.py

```diff
@@ -73,10 +73,13 @@ def __init__(self, estimator, logdir=None):
     @staticmethod
     def _cmd_exists(cmd):
```

Comment:
out of scope, but thought I'd mention it for future reference - it looks like there are better ways to implement this function 😂: https://stackoverflow.com/a/34177358. Maybe we'll look into that Python 3.3+ version when we drop Python 2.7 support.

Reply:
Looks like ``shutil`` does not have ``which`` module in python 2.7.

So we can adopt this change when we drop python 2.7 support.